rails-commenting.rb

```diff
@@ -71,6 +71,6 @@ def blog_post_params
 # FILE: app/models/blog_post.rb
 
 class BlogPost < ApplicationRecord
-  # 10)
+  # 10) has_many is giving the blog the ability to have comments in the scope of the application.
```

Comment:
It builds a has_many and belongs_to relationship between BlogPost and comments where each blogpost can contain many comments.

Good job on these Matt! They really show understanding of a full-stack app!